lib/ansible/utils/module_docs_fragments/files.py

```diff
@@ -66,9 +66,10 @@ class ModuleDocFragment(object):
     description:
       -  Normally this module uses atomic operations to prevent data corruption or inconsistent reads from the target files,
          sometimes systems are configured or just broken in ways that prevent this. One example are docker mounted files,
-         they cannot be updated atomically and can only be done in an unsafe manner.
+         they cannot be updated atomically (from inside the container) and can only be written in an unsafe manner.
       -  This boolean option allows ansible to fall back to unsafe methods of updating files for those cases in which you do
-         not have any other choice. Be aware that this is subject to race conditions and can lead to data corruption.
+         not have any other choice (however, it doesn't actually force Ansible to do anything). Be aware that this is subject
```

Comment:
I have my doubts too, but I'm also still in the dark about what exactly this option does, only know what it _does not_ do.

"Allows to fall back" suggests that I could use it to make Ansible drop the atomic file operation. As #24449 indicates, the reality is not quite obvious and further explanation is in order.